image/image.go

```diff
@@ -174,6 +180,17 @@ func (img *Image) OperatingSystem() string {
 	return os
 }
 
+// Platform generates an OCI platform from the image
+func (img *Image) Platform() ocispec.Platform {
+	return ocispec.Platform{
+		Architecture: img.BaseImgArch(),
+		Variant:      img.BaseImgVariant(),
+		OS:           img.OperatingSystem(),
+		OSVersion:    img.OSVersion,
+		OSFeatures:   img.OSFeatures,
+	}
+}
```

Comment:
Since the primary use case of this is for exporting the image as it exists in our data store, shouldn't we instead export it as-is, _not_ normalized and let users/clients/tools normalize it on import if they so desire?

Reply:
Yes, I'd be perfectly fine with just using the raw values as a platform;


```go
// Platform generates an OCI platform from the image. Fields are returned
// as-is, without calling [platforms.Normalize].
func (img *Image) Platform() ocispec.Platform {
	return ocispec.Platform{
		Architecture: img.Architecture,
		OS:           img.OS,
		OSVersion:    img.OSVersion,
		OSFeatures:   img.OSFeatures,
		Variant:      img.Variant,
	}
}
```

Reply:
Updated